jib-core/src/test/java/com/google/cloud/tools/jib/image/ReproducibleLayerBuilderTest.java

```diff
@@ -246,26 +246,26 @@ public void testBuild_parentDirBehavior() throws IOException {
       Assert.assertEquals(Instant.ofEpochSecond(1), root.getModTime().toInstant());
 
       // parentAAA (custom permissions, custom timestamp)
-      TarArchiveEntry rootParentAAA = in.getNextTarEntry();
-      Assert.assertEquals(040111, rootParentAAA.getMode());
-      Assert.assertEquals(Instant.ofEpochSecond(10), rootParentAAA.getModTime().toInstant());
+      TarArchiveEntry rootParentAaa = in.getNextTarEntry();
```

Comment:
Maybe we should just find a better name for the variables in this test.

Reply:
Renamed.